service_container/tags.rst

```diff
@@ -16,8 +16,7 @@ to be used for a specific purpose. Take the following example:
 
         # app/config/services.yml
         services:
-            app.twig_extension:
-                class: AppBundle\Twig\AppExtension
+            AppBundle\Twig\AppExtension:
                 public: false
```

Comment:
I'd say, let's not add the "defaults" until we decide what to do with that new option.